playground/mocks/data/idp/idx/authenticator-enroll-ov-device-bootstrap-multiple-devices.json

```diff
@@ -176,7 +176,7 @@
                     ]
                   }
                 },
-                "relatesTo": "$.authenticators.value[3]"
+                "relatesTo": "$.authenticators.value[2]"
```

Comment:
Were these `relatesTo` values in the mocks incorrect?

Reply:
Yes. This causes `authClient.idx.proceed/start` to throw exception